### PR TITLE
Write test and fix some conversion bugs

### PR DESCRIPTION
**Pull request by dtan4**
## WHY

Currently we have no wayto verify whether `pq2gorm` generates gorm models correctly. In addition, I found conversion bugs with some field types.
## WHAT

Write test to verify whether generated files are correspond to expected model files. Enable [Travis CI](https://travis-ci.org/wantedly/pq2gorm) too. To pass these tests, I made some fix around conversion.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -28,6 +28,19 @@ endif
 install:
 	go install $(LDFLAGS)
 
+.PHONY: test
+test:
+	@rm -rf out
+	@docker-compose stop > /dev/null
+	@docker-compose rm -f > /dev/null
+	docker-compose up -d db
+	sleep 5
```
Comment (koudaiii) [inline Makefile]:
```
$ make test
docker-compose up -d db
Creating network "pq2gorm_default" with the default driver
Creating pq2gorm_db_1
sleep 5
docker-compose exec db psql -U postgres -d test -f /testdata/db.dump
psql: could not connect to server: No such file or directory
    Is the server running locally and accepting
    connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
make: *** [test] Error 2
```

uhmmm.

Could you change time from `5s` to `10s` ?

Comment (dtan4) [reply Makefile]: https://github.com/wantedly/pq2gorm/pull/4/commits/276f62a893955977baffd0fc1cd2e515eeb5b118

Changed to wait until connection established (max 25s).

**Comment by koudaiii** [comment]
Great Job!!!

**Comment by koudaiii** [comment]
LGTM!!

**Pull request merged**